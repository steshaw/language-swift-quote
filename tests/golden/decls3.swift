let welcomeMessage: String = "Hello"
print(welcomeMessage)
