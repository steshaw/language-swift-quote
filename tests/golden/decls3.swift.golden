"<stdin>" (line 1, column 19):
unexpected ':'
expecting letter or digit, "<", "`", ".", "(", "[", "=", "is", "as", ",", ";", "import", "class", "convenience", "dynamic", "final", "infix", "lazy", "mutating", "nonmutating", "optional", "override", "postfix", "prefix", "required", "static", "unowned", "weak", "internal", "private", "public", "let", "var", "typealias", "func", "try", "try?", "try!", identifier, float, integer, literal string, "true", "false", "nil", "__FILE__", "__LINE__", "__COLUMN__", "__FUNCTION__", "self", "<superclass-expression>", "{", "<implicit-member-expression>", "_", "for", "while", "repeat", "if", "guard", "switch", "return" or end of input
attribute not implemented